plugins/aac-gen-design-doc/aac_gen_design_doc_impl.py

```diff
@@ -0,0 +1,108 @@
+"""AaC Plugin implementation module for the aac-gen-design-doc plugin."""
+
+import os
+import sys
+from functools import partial
+
+from iteration_utilities import flatten
+from jinja2 import Environment, FileSystemLoader
+
+from aac import parser, util
+
+plugin_version = "0.0.1"
+
+
+def gen_design_doc(template_file: str, architecture_files: str, output_directory: str):
+    """
+    Generate a System Design Document from Architecture-as-Code models.
+
+    Args:
+        `template_file` <str>: The name of the template file to use for generating the document.
+        `architecture_files` <str>: A comma-separated list of yaml file(s) containing the modeled
+                                      system for which to generate the System Design document.
+        `output_directory` <str>: The directory to which the System Design document will be written.
+    """
```

Comment:
@Coffee2Bits 

The reason I did it like this is because this is how it shows up in my editor (which uses LSP, like VSCode) so I assumed it would show up in VSCode for you guys the same way - below is the difference.

![image](https://user-images.githubusercontent.com/2924081/140423476-f4391407-18b0-410d-9b8e-20bc556ac736.png)

Using the official Google style would show up like this:

![image](https://user-images.githubusercontent.com/2924081/140425340-d183bcb3-1bb6-43c1-86de-780e07a054d9.png)

Of course, I'm not too worried about keeping that formatting if we want it to be the other way, it just makes it easier to distinguish variables, types, etc from text.